dev/PushNotifications/PushNotificationsLongRunningTask.ProxyStub/NotificationsLongRunningProcess.idl

```diff
@@ -38,13 +38,4 @@ interface INotificationsLongRunningPlatform : IUnknown
     HRESULT RemoveToastRegistrationMapping([in] LPCWSTR processName);
 };
 
-[uuid(PUSHNOTIFICATIONS_LIBID_UUID)]
-[version(1.0)]
-library NotificationsLongRunningPlatformLibrary
-{
-    [uuid(PUSHNOTIFICATIONS_IMPL_CLSID_UUID)]
-    coclass NotificationsLongRunningPlatform
-    {
-        [default] interface INotificationsLongRunningPlatform;
-    };
-};
+declare_guid(CLSID_NotificationsLongRunningPlatform, PUSHNOTIFICATIONS_IMPL_CLSID_UUID);
```

Comment:
Interesting. Can you explain why this was changed?

Reply:
For my understanding :)

Reply:
I'd generally consider using library blocks just to put a CLSID in the header (and uuid lib) to be a minor anti-pattern. We're not actually interested in generating a typelib here, so there's no real functional advantage to the library block.

Arguably it's providing some extra value by describing the interfaces impelemented on the CLSID, but 1) there's nothing enforcing/requiring the interface list to be accurate, so it has no more weight than a comment; and 2) in simple cases like this, convention strongly suggests that CLSID_NotificationsLongRunningPlatform probably implements INotificationsLongRunningPlatform.